Sdtrig.tex

```diff
@@ -319,6 +319,37 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+For address matches without a mask, \RcsrTdataTwo must be able to hold all valid
+addresses in all supported translation modes. That means that after writing any
+of these valid addresses, the exact same value is read back, including any high
+bits.
+
+\begin{commentary}
+    If physical addresses are less than XLEN bits wide, they are zero-extended.
+    If virtual addresses are less than XLEN bits wide, they are sign-extended.
+    Be sure to have enough flops in \RcsrTdataTwo to hold all XLEN bits of the
+    full range of both physical and virtual addresses.
+\end{commentary}
+
+If \RcsrTdataTwo can hold any invalid addresses, then writes of an
+invalid address that can not be represented as-is should be converted to
+a different invalid address that can be represented.
+
+For invalid instruction fetch and load and store effective addresses,
+the compare value may be changed to a different invalid address.
+
+\begin{commentary}
+    A straightforward trigger implementation would compare the instruction fetch
+    and load and store effective addresses to the contents of \RcsrTdataTwo.
+    This is most intuitive to a user setting triggers. Here we accommodate
+    implementations that internally shorten invalid addresses.
+\end{commentary}
+
+In addition, an implementation may choose to inhibit all trigger matching
+against invalid addresses.
```

Comment:
... especially if there is no support for storage of any invalid address values in tdata2.

Reply:
Done.